cmd/antrea-agent/agent.go

```diff
@@ -418,11 +422,24 @@ func run(o *Options) error {
 
 	// Start the Antrea IPAM agent.
 	if features.DefaultFeatureGate.Enabled(features.AntreaIPAM) {
+		listOptions := func(options *metav1.ListOptions) {
+			options.FieldSelector = fields.OneTermEqualSelector("spec.nodeName", nodeConfig.Name).String()
+		}
+		podInformer := coreinformers.NewFilteredPodInformer(
```

Comment:
Could we use informerFactory?

And I feel better to put these lines in InitializeAntreaIPAMController(). You can refer to InitializeNPLAgent().

Reply:
Added `localNodeInformerFactory` here.
I think we should construct/start `localNodeInformerFactory` at `agent.go` and generate resource informer inside each feature controller initialization, as what `informerFactory` did.